tests/pod.py

```diff
@@ -0,0 +1,69 @@
+import yaml
+import time
+
+from os import path
+from kubernetes import client
+from pytest_testconfig import config as testconfig
+from kubernetes.client.rest import ApiException
+
+
+def wait_for_pod_to_be_ready(pod_name, name_space, time_out=None):
+    total_sleep_time = 0
+    while True:
+        resp = client.CoreV1Api().read_namespaced_pod(name=pod_name, namespace=name_space)
```

Comment:
can this return error? what should happen if so?

Reply:
I don't know. I havn't met an error here.
If we see that there are errors here we will handle them in the future